spec/sample-data/valid/control-statement-02.json

```diff
@@ -56,6 +63,5 @@
         }
       }
     }
-  ],
-  "control_level": "unknown"
```

Comment:
This field is also duplicated in `control-statement-01.json`, so will fix there too. And will also extract into separate commit.